src/Data/Quote/QuoteDataClass.ts

```diff
@@ -6,272 +6,350 @@ export const Quote = provideLocalStorageDataClass('Quote', {
       keyword: 'Kompressoranlage 400 bar',
       number: '00097',
-      type: 'Re-equip machine',
+      type: 'PSA_QUO_STY_NEW',
+      typeLocalized: 'Re-equip machine',
       customer: 'Maier Schmiertechnik GmbH (München)',
       commercialAgent: 'F87BDC400E41D630E030A8C00D01158A',
       technicalAgent: '5C6EDDBA2459CAB4E040007F0100275E',
       salesPartner: '',
-      status: 'quote preparation',
-      mainStatus: 'Quote generation',
+      status: 'PSA_PRO_QUO_WRK',
+      statusLocalized: 'quote preparation',
+      mainStatus: 'PSA_SAP_MAI_QUO_ORD_CER',
+      mainStatusLocalized: 'Quote generation',
       open: true,
       quoteAt: '2023-05-03T00:00:00Z',
       orderAt: '2023-05-03T00:00:00Z',
       deliveryAt: '2023-05-27T00:00:00Z',
-      termsOfDelivery: 'cost and freight',
-      termsOfPayment: 'Payment target (ERP 40)',
+      termsOfDelivery: 'RFC_TOD_CFR',
+      termsOfDeliveryLocalized: 'cost and freight',
+      termsOfPayment: 'PSI_TOP_40',
+      termsOfPaymentLocalized: 'Payment target (ERP 40)',
       totalPrice: 67417.14285714284,
+      totalPriceCurrency: 'EUR',
       description: '',
       validUntil: null,
     },
     {
       _id: '81282658BDCF471FB2ED3051D6528597',
       keyword: 'Angebot Schmieranlage für Maier',
       number: 'P-20-009128',
-      type: 'Standard',
+      type: 'PSA_QUO_STY_STD',
+      typeLocalized: 'Standard',
       customer: 'Maier Schmiertechnik GmbH (München)',
       commercialAgent: '052601BEBCEC39C8E040A8C00D0107AC',
       technicalAgent: '18EEEAEC56D37397E040A8C00F012319',
       salesPartner: 'Hono Gerätebau GmbH (Grafing)',
-      status: 'quote won and qualified to order',
-      mainStatus: 'Offer won',
+      status: 'PSA_PRO_QUO_WON',
+      statusLocalized: 'quote won and qualified to order',
+      mainStatus: 'PSA_SAP_MAI_QUO_ORD_CRE',
+      mainStatusLocalized: 'Offer won',
       open: false,
       quoteAt: '2023-12-10T12:48:42Z',
       orderAt: '2023-12-10T13:05:58Z',
       deliveryAt: null,
-      termsOfDelivery: 'cost and freight',
+      termsOfDelivery: 'RFC_TOD_CFR',
+      termsOfDeliveryLocalized: 'cost and freight',
       termsOfPayment: '',
+      termsOfPaymentLocalized: '',
       totalPrice: 25657.904761904745,
+      totalPriceCurrency: 'EUR',
       description: '',
       validUntil: null,
     },
     {
       _id: 'C6D30A8964AF42C795529F1CCEB3484E',
       keyword: 'Angebot Max 40',
       number: 'P-18-009104',
-      type: 'Standard',
+      type: 'PSA_QUO_STY_STD',
+      typeLocalized: 'Standard',
       customer: 'Maier Schmiertechnik GmbH (München)',
       commercialAgent: '052601BEBCEC39C8E040A8C00D0107AC',
       technicalAgent: 'F87BDC400E41D630E030A8C00D01158A',
       salesPartner: '',
-      status: 'quote lost',
-      mainStatus: 'Quote lost',
+      status: 'PSA_PRO_QUO_LST',
+      statusLocalized: 'quote lost',
+      mainStatus: 'PSA_SAP_MAI_QUO_LST',
+      mainStatusLocalized: 'Quote lost',
```

Comment:
* [ ] FYI lower/upper case inconsistency (was already present before)

Reply:
I'm not sure, what you mean. Do you men `mainStatus` is wrongly cases? Should it be `mainstatus` instead?

Reply:
statusLocalized: '**q**uote lost',
mainStatusLocalized: '**Q**uote lost',